features/update.feature

```diff
@@ -922,3 +922,36 @@ Feature: update
     When I run `msync update -m "Update Gemfile"`
     Then the exit status should be 0
     Then the output should contain "Using repository's default branch: develop"
+
+  Scenario: Adding a new file from a template using meta data
+    And a file named "config_defaults.yml" with:
+      """
+      ---
+      """
+    Given a file named "managed_modules.yml" with:
+      """
+      ---
+        - puppet-test
+      """
+    And a file named "modulesync.yml" with:
+      """
+      ---
+        namespace: maestrodev
+        git_base: https://github.com/
+      """
+    And a directory named "moduleroot"
+    And a file named "moduleroot/test.erb" with:
```

Comment:
Nit: having this named `test.erb` meant that there's a lot of `test` going on. I'd suggest naming this `README.erb` or similar to clearly distinguish from the module name (`puppet-test`).

Reply:
I just adopted a generic test file, in my use-case I use the metadata to build a project specific `.gitlab-ci.yml`.

Reply:
Did you forget to push?

Reply:
No, I wanted my explain my intention to create the feature.

Since other tests used `test.erb` I continued it here.